CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+0.41.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.41.0
```

Comment:
I just merged #620, which would be a good addition to this release. Could you please update the changelog?